src/views/LocusGeneDiseaseView.vue

```diff
@@ -410,20 +410,15 @@ export default {
               <h6>Mechanism</h6>
             </td>
             <td class="w-75">
-              <p
-                v-if="
-                  locusGeneDiseaseData.molecular_mechanism &&
-                  locusGeneDiseaseData.molecular_mechanism.length > 0
-                "
-              >
-                {{ locusGeneDiseaseData.molecular_mechanism[0].mechanism }}
+              <p v-if="locusGeneDiseaseData.molecular_mechanism">
+                {{ locusGeneDiseaseData.molecular_mechanism.mechanism }}
```

Comment:
Pls apply similar changes for mechanism categorisation and categorisation support

Reply:
Done, also add changes to evidence